__tests__/app.test.js

```diff
@@ -27,6 +27,36 @@ describe("app", () => {
         });
     });
   });
+  describe("/api", () => {
+    test("200: responds with a status of 200", () => {
+      return request(app).get("/api").expect(200);
+    });
+    test("200: responds with a json object of endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+          const { body } = response;
+          expect(typeof body).toBe("object");
+          const bodyArr = Object.entries(body);
+          bodyArr.forEach((endpointArr) => {
```

Comment:
When using iterating through a an object to do tests on each element, it's important to check the length is greater than 0, because tests that don't get a chance to run count as if they've passed - an empty array would pass these tests as is